src/Playwright/build/Microsoft.Playwright.targets

```diff
@@ -18,6 +19,13 @@
       <_PlaywrightCopyItems Include="$(MSBuildThisFileDirectory)..\.playwright\node\linux-arm64\**" Condition="$(PlaywrightPlatform.Contains('linux-arm64'))">
         <PlaywrightFolder>node\linux-arm64\</PlaywrightFolder>
       </_PlaywrightCopyItems>
+      <!-- 'osx' to stay backwards compatible -->
+      <_PlaywrightCopyItems Include="$(MSBuildThisFileDirectory)..\.playwright\node\mac-x64\**" Condition="$(PlaywrightPlatform.Contains('osx')) OR $(PlaywrightPlatform.Contains('osx-x64'))">
+        <PlaywrightFolder>node\mac-x64\</PlaywrightFolder>
```

Comment:
The folder in the nuget package is actually named `darwin-x64` which was correctly done in the csproj but not in this file. See #2531